### ISSUE TITLE
Disablelogin and Disablequit Permissions

### ISSUE THREAD

**Issue by k0nker** [Zrips/CMI#732]
If `cmi.messages.disablelogin` or `cmi.messages.disablequit` permissions are negated within the server permissions plugin for a player it acts like it's active for the player and doesn't send join/quit messages.  Essentially setting the permission to false is acting like it's set to true, but if you leave it undefined for the player it doesn't interfere and the message goes through.

**CONFIG SECTION :**
```
Messages:
  Login:
    # If set to true, login message wont be shown
    Disabled: false
    # Defines number of players from which to automatically start hiding join messages
    # Set to -1 to disable this
    AutoHideFrom: -1
    Custom:
      # If set to true, custom login message will be used. cmi.messages.disablelogin can be used to disable message for player
      Use: false
  Logout:
    # If set to true, logout message wont be shown
    Disabled: false
    # Defines number of players from which to automatically start hiding logout messages
    # Set to -1 to disable this
    AutoHideFrom: -1
    Custom:
      # If set to true, custom logout message will be used. cmi.messages.disablequit can be used to disable message for player
      Use: false
  # Check locale file for translation and custom placeholders: [playername], [totalUsers], [onlinePlayers]
```

**Cmi Version (using`/cmi version`): CMI plugin version: 8.0.4.5** 

**Server Version (using `/ver`): Paper version git-Paper-103 (MC: 1.13)**

**Comment by k0nker** [comment]
This is also happening with cmi.keepinventory. If the permission is negated it's acting like it's true.

**Comment by maartenvr98** [comment]
Same problem for me

**Comment by CloudeLecaw** [comment]
i also registered the same behaviro with some permissions

**Comment by MCTyler** [comment]
Is this just with Pex or LuckPerms too? ; or what Perms? I think this info would help @Zrips

**Comment by LogGits** [comment]
Has this issue been resolved in `8.0.15.0`?

**Comment by Zrips** [comment]
This should be resolved in latest builds.
